src/Html.php

```diff
@@ -692,55 +639,64 @@ public static function textarea($name, $value = '', $options = []): string
      */
-    public static function checkbox($name, $checked = false, $options = []): string
+    public static function checkbox(string $name, bool $checked = false, array $options = []): string
     {
         return static::booleanInput('checkbox', $name, $checked, $options);
     }
 
     /**
      * Generates a boolean input.
+     *
      * @param string $type the input type. This can be either `radio` or `checkbox`.
      * @param string $name the name attribute.
      * @param bool $checked whether the checkbox should be checked.
      * @param array $options the tag options in terms of name-value pairs. The following options are specially handled:
      *
-     * - uncheck: string, the value associated with the uncheck state of the checkbox. When this attribute
-     *   is present, a hidden input will be generated so that if the checkbox is not checked and is submitted,
-     *   the value of this attribute will still be submitted to the server via the hidden input.
-     * - label: string, a label displayed next to the checkbox.  It will NOT be HTML-encoded. Therefore you can pass
-     *   in HTML code such as an image tag. If this is is coming from end users, you should {@see encode()} it to prevent XSS attacks.
+     * - uncheck: string, the value associated with the uncheck state of the checkbox. When this attribute is present,
+     *   a hidden input will be generated so that if the checkbox is not checked and is submitted, the value of this
+     *   attribute will still be submitted to the server via the hidden input.
+     * - label: string, a label displayed next to the checkbox. It will NOT be HTML-encoded. Therefore you can pass in
+     *   HTML code such as an image tag. If this is is coming from end users, you should {@see encode()} it to prevent
+     *   XSS attacks.
      *   When this option is specified, the checkbox will be enclosed by a label tag.
-     * - labelOptions: array, the HTML attributes for the label tag. Do not set this option unless you set the "label" option.
+     * - labelOptions: array, the HTML attributes for the label tag. Do not set this option unless you set the "label"
+     *   option.
      *
-     * The rest of the options will be rendered as the attributes of the resulting checkbox tag. The values will
-     * be HTML-encoded using {@see encode()}. If a value is null, the corresponding attribute will not be rendered.
+     * The rest of the options will be rendered as the attributes of the resulting checkbox tag. The values will be
+     * HTML-encoded using {@see encode()}. If a value is null, the corresponding attribute will not be rendered.
      * See {@see renderTagAttributes()} for details on how attributes are being rendered.
      *
      * @return string the generated checkbox tag
      */
-    private static function booleanInput($type, $name, $checked = false, $options = []): string
-    {
+    protected static function booleanInput(
```

Comment:
It was OK before.

Reply:
PSR-2 max 120 characters for line,  like the new PSR-12.

Reply:
There is no hard line limit in PSR-2 nor PSR-12.